### DISCUSSION TITLE
Favoring consistent spelling over 100% accurate pronunciation

### DISCUSSION THREAD

**Discussion by trosel** [Shavian-info/readlex#32] [Spelling principles]
https://github.com/Shavian-info/readlex/issues/29

I mentioned how a spelling standard that is sufficiently global should include both final R and yods. Shavian includes compound letters for both of these things already.

If yods are retained, Americans (for example) would need to be taught that in some words, they just don't pronounce the yod (that is their choice), but they should keep the yod in spelling for the sake of etymological clarity, among other things.

For example, dew vs do can be distinguished in spelling if yods are kept.

**Comment by Shavian-info** [comment]
In transliterations I favour the pronunciations with yods. But I include the yod-less pronunciations in the Read Lexicon following the advice of Kingsley Read himself. My goal with the Read Lexicon is not really to enforce a single standard, but to have a standard that provides a sort of 'theme' for people's personal variations. The approach you suggest (if it means telling people not to use the yod-less option at all) is a bit more prescriptive than I'm comfortable with.

**Comment by trosel** [comment]
It's less about pronunciation and more about holding together multiple different accents, ironically. The idea is that the yod would be a visual signal to pronounce it the way that they are used to pronouncing it (some with "ch", some with "yoo", some with "oo"). That way you don't have to have 3 different spellings of choon, dyoon, and doon.
